hr_holidays_validator_list/models/hr_employee.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class HrEmployeeBase(models.AbstractModel):
+    _inherit = "hr.employee.base"
+
+    leave_manager_ids = fields.Many2many(
+        "res.users",
+        string="Time Off Managers",
+        store=True,
+        readonly=False,
+        help="""Select the users responsible for approving 'Time Off' of this employee.\n
+        If empty, the approval is done by an Administrator
+        or Approver (determined in settings/users).""",
+    )
+
+    def _set_leave_manager_id_from_leave_manager_ids(self, values):
```

Comment:
You should have used a stored readable only computed method on leave_manager_id with a depends on leave_manager_ids to propagate any change.

Reply:
You're right I am going to make the change

Reply:
@StephaneMangin The field `leave_manager_ids` is already a computed stored field. Question is can we insert nicely the set of the values from the M2M nicely or not. We might want to keep leave_manager_ids value to it's current natural computation.

@victorvermot 

What we could probably do to improve the code is:

Override https://github.com/odoo/odoo/blob/17.0/addons/hr_holidays/models/hr_employee.py#L152-L159
this way:

``` python
 @api.depends('parent_id', 'leave_manager_ids')
    def _compute_leave_manager(self):
            
        for employee in self:
            if self.leave_manager_ids:
               self.leave_manager_id = leave_manager_ids[0]     
           else:
                super()._compute_leave_manager()
```